src/main/resources/org/biouno/unochoice/DynamicReferenceParameter/index.jelly

```diff
@@ -112,14 +112,14 @@
             for (var j = 0; j &lt; divs.length ; j++) {
                 var div = divs[j];
                 var hiddenNames = jQuery(div).find('input[name="name"]');
-                if (hiddenNames[0].value == referencedParameters[i]) {
+                if (hiddenNames[0].value === referencedParameters[i]) {
                     var children = div.children;
                     for (var k = 0; k &lt; children.length; ++k) {
                         var child = children[k];
-                        if (child.getAttribute('name') == 'value') {
+                        if (child.getAttribute('name') === 'value') {
                             parameterElement = child;
                             break;
-                        } else if (child.tagName == 'DIV') {
+                        } else if (child.tagName === 'DIV' || child.tagName === 'SPAN') {
```

Comment:
You can ignore the `==` replaced by `===`, that's just JS.

The change above including the `SPAN` tag is the actual fix. The new Jenkins UI wraps the checkbox in a SPAN instead of using a DIV.